service/common/src/main/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalogHandler.java

```diff
@@ -331,7 +337,17 @@ public LoadTableResponse createTableDirect(Namespace namespace, CreateTableReque
     if (isStaticFacade(catalog)) {
       throw new BadRequestException("Cannot create table on static-facade external catalogs.");
     }
-    return CatalogHandlers.createTable(baseCatalog, namespace, request);
+    // TODO check to see if we will need do this in createTableWriteDelegation, createTableStaged,
```

Comment:
Yes, we do. Also, this needs to be feature flagged off.

Reply:
Will add this, also was wondering if I can get more clarity on what you mean by flagged off?